### ISSUE TITLE
Add condensed 5-day forecast

### ISSUE THREAD

**Issue by bhe69** [martykan/forecastie#497]
I like forecastie for its design and its accuracy, but I'd like to see a condensed 5-day forecast showing only the important data instead of scrolling through the hourly forecasts of several days.

Openweathermap offers the function "daily forecast" via API: [https://openweathermap.org/api/one-call-api](https://openweathermap.org/api/one-call-api)

The result could contain: 
Date, Min/Max Temperature, Weather condition (sunny, overcast etc.), Probability of precipitation, Precipitation volume, wind speed.

See also a screenshot from a proprietary weather provider how it could look like:
![grafik](https://user-images.githubusercontent.com/35567477/88491522-95ea4a80-cfa3-11ea-95f2-cf764a3ffcfe.png)

**Comment by robinpaulson** [comment]
yes, this does look useful, similar in intent to the "notification" requests #19 

also: do try the graphs, they may be more useful as well.
